uportal-war/src/main/java/org/apereo/portal/shell/PortalShell.java

```diff
@@ -76,9 +74,6 @@ public static void main(String[] args) throws Exception {
                         new CompilerConfiguration(System.getProperties());
                 final GroovyShell shell = new GroovyShell(binding, conf);
                 shell.run(scriptFile, args);
-            } else {
```

Comment:
Yes, I would like to keep it. Think about the scenario tomorrow somebody forget to/does not pass script parameter, we do not want to start Groovy shell.